README.md

```diff
@@ -1,8 +1,50 @@
-About openfe-feedstock
-======================
+About openfe-split-feedstock
+============================
 
 Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/openfe-feedstock/blob/main/LICENSE.txt)
 
+
+About openfe-split
+------------------
+
+Home: https://github.com/OpenFreeEnergy/openfe
+
+Package license: MIT
+
+Summary: Free software for Free Energies.
+
+Development: https://github.com/OpenFreeEnergy/openfe
+
+Documentation: https://openfe.rtfd.io/
```

Comment:
Should this not point to docs.openfree.energy?

Reply:
good catch!

Reply:
Also I think `home` should be `https://openfree.energy/`